src/array/__tests__/arrayEqual-test.js

```diff
@@ -0,0 +1,18 @@
+import arrayEqual from '../arrayEqual';
+import { expect } from 'chai';
+
+describe('arrayEqual', () => {
```

Comment:
Be more thorough.
What happens when there are no elements in the list?
What happens when the lists are different length? Like [1, 2, 3] and [1, 2, 3, 4]
Different types [1, 2, 3], ['1', '2', '3']